bagger/scripts/test.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+from pathlib import Path
+from redata.commons import logger
+
+from bagger.bag import Bagger, Status
+from bagger.config import get_args, TOMLDecodeError
+
+
+def main() -> None:
+    path = 'TestBags/7873476_01_Jeffrey_C_Oliver_b3e8ca9fce68c6dd95c4b71efe9220b7'
```

Comment:
If not too big, highly recommend including the test data in version control (assuming there is no sensitive info. If there is, can use dummy files)